media-sound/nulloy/nulloy-9999.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3 xdg
+
+DESCRIPTION="Music player with a waveform progress bar (sound visualization like in audio editors)"
+HOMEPAGE="https://nulloy.com"
+
+EGIT_REPO_URI="https://github.com/nulloy/nulloy"
+EGIT_CLONE_TYPE="shallow"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+skins"
+
+BDEPEND="dev-qt/linguist-tools"
+
+DEPEND="
+	dev-qt/qtcore
+	dev-qt/designer
+	dev-qt/linguist
+	dev-qt/qtscript
+	dev-qt/qtx11extras
+	media-libs/gstreamer
+	media-libs/gst-plugins-base
+	media-plugins/gst-plugins-meta
+"
+RDEPEND="${DEPEND}"
+
+src_unpack() {
+	git-r3_src_unpack
+
+	if use skins ; then
+		EGIT_REPO_URI=https://gitlab.com/vitaly-zdanevich/nulloy-theme-night.git
+		EGIT_CHECKOUT_DIR=${WORKDIR}/night
+		git-r3_src_unpack
+	fi
+}
+
+src_prepare() {
+	if use skins ; then
+		default
+
+		eapply "${FILESDIR}"/nulloy.patch
+		cp -r $WORKDIR/night src/skins
+	fi
+}
+
+src_configure() {
+	# Upstream ./configure script does not support specifying an option's
+	# value after an equal sign like in '--prefix="${EPREFIX}/usr"', so we
+	# have to set up all the options ourselves and call the script directly
+	local myconfargs=(
+		$(use skins || echo --no-skins)
+		--no-update-check
+		--no-taglib
+		--gstreamer-tagreader
```

Comment:
Thank you for the effort, however I would recommend using taglib plugin, as gstreamer-tagreader feature list is lacking behind.

Reply:
What features do you mean? I do not want to add additional dependency :(

Reply:
Cover art feature is taglib only. Also, I suspect you are building taglib already, since gstreamer uses it as dependency.

Reply:
But I have cover art with this ebuild, unused dependencies are removed from my system.

Reply:
>But I have cover art with this ebuild

Hm, that's very strange, `--no-taglib` should disable cover art support.

Reply:
![image](https://user-images.githubusercontent.com/3514015/160212828-902709b8-4d9c-48d0-bd09-94b74318d37f.png)

Reply:
Maybe I have a bug in `configure` script. Is there a build log?

Reply:
Here https://github.com/nulloy/nulloy/issues/196 was a build log, or do you need a successful one?